warehouse/scripts/push_to_metabase.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -e
+
+dbt-metabase models \
+    --dbt_manifest_path ./target/manifest.json \
+    --dbt_database cal-itp-data-infra \
+    --metabase_host dashboards.calitp.org \
+    --metabase_user $METABASE_USER \
+    --metabase_password $METABASE_PASSWORD \
+    --metabase_database "(Internal) Staging Warehouse Views"
```

Comment:
Do we want this hard coded? I guess this is for testing?

Reply:
Oh yeah this was just for testing, it's not that useful actually. Will delete to avoid confusion.